llvm/test/Transforms/MemCpyOpt/memcpy-gep-position-guard.ll

```diff
@@ -0,0 +1,51 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
```

Comment:
I cannot reproduce the crash `LLVM ERROR: Function @test_gep_modified changed by MemCpyOptPass without invalidating analyses` on the trunk build.

Reply:
I add '-DLLVM_ENABLE_EXPENSIVE_CHECKS=true' to my cmake, this might cause the ERROR.